3rd-party/extensions/hotkeys/js/jquery.fancytree.hotkeys.js

```diff
@@ -15,9 +15,11 @@
 	var initHotkeys = function(tree, data) {
 		$.each(data, function(event, keys) {
 			$.each(keys, function(key, handler) {
-				$(tree.$container).on(event, null, key, function() {
+				$(tree.$container).on(event, null, key, function(evt) {
 					var node = tree.getActiveNode();
 					handler(node);
+					evt.preventDefault();
```

Comment:
Ok, if you update your PR (changing the line to `return handler(node, evt);` and remove `evt.preventDefault()`), I will merge :)
Feel free to add a comment to the example as well.